CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ option(
   "Build documentation without building NEURON. It will be assumed that import neuron works, nrnivmodl is in PATH, etc."
   ${NRN_ENABLE_DOCS_WITH_EXTERNAL_INSTALLATION_DEFAULT})
 mark_as_advanced(NRN_ENABLE_DOCS_WITH_EXTERNAL_INSTALLATION)
+option(NRN_ENABLE_DOCS_NMODL "Enable documentation for NMODL" ${NRN_ENABLE_DOCS_NMODL_DEFAULT})
```

Comment:
This extra flag is needed because `NRN_ENABLE_CORENEURON` enables too much (it builds coreNEURON when we don't need it), while `NRN_ENABLE_NMODL` actually replaces NOCMODL, which is _not_ what we want at the moment. This flag basically enables only building of NMODL, and nothing else (maybe I could rename it to `NRN_BUILD_NMODL`?).